float64 timestamp_slam
geometry_msgs/PoseStamped Keyframe
